.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+**Helpful additional information**
+(*Add X inside the boxes or click them after submitting ticket*)
+- [ ] Did VisIt crash?
+- [ ] Did you get wrong results?
+
+**Severity**
```

Comment:
Would you mind updating heading to be **Impact** and adjust the text for check boxes to match the adjusted lable descriptions

Reply:
Done. I also added an impact checklist to the feature request.